app/scripts/filters/resources.js

```diff
@@ -1322,8 +1322,8 @@ angular.module('openshiftConsole')
       return cmd;
     };
 
-    return function(container, image) {
-      if (!container || !image) {
+    return function(container, image, returnArgsIfNoCmd) {
```

Comment:
Originally I wasn't adding `<image-entrypoint>` when we didn't have the image and it was only returning the args so I didn't want it to change the behavior on debug terminal. But yeah now I think I can drop the arg and just always have that behavior.